routes/user.js

```diff
@@ -221,21 +206,14 @@ router.route('/forgot')
                 });
             },
             function(token, user, done) {
-                var smtpTransport = nodemailer.createTransport('SMTP', {
-                    service: 'SendGrid',
-                    auth: {
-                        user: process.env.SENDGRID_USER || config.get('emailVerification.user'),
-                        pass: process.env.SENDGRID_PASS || config.get('emailVerification.pass')
-                    }
-                });
                 var mailOptions = {
                     to: user.email,
                     from: 'noreply@theotech.org',
                     subject: 'Reset Password',
-                    text: 'You are receiving this because you (or someone else) have requested the reset of the password for your account.\n\n' +
-                        'Please click on the following link, or paste this into your browser to complete the process:\n\n' +
-                        'http://' + req.headers.host + '/user/reset/' + token + '\n\n' +
-                        'If you did not request this, please ignore this email and your password will remain unchanged.\n'
+                    text: 'Someone has requested to reset the password for your account at glorify.cc\n\n' +
+                        'If you did not request this, you can ignore this message.\n\n' +
+                        'Otherwise, please follow this link to reset your password:\n' +
+                        'http://' + req.headers.host + '/user/reset/' + token
```

Comment:
recommend using https: 
I know that in NODE_ENV = development, it needs to be http:, but for production we should use https.